src/Components/RegisterToVote/RegisterToVote.js

```diff
@@ -0,0 +1,18 @@
+import './RegisterToVote.css';
+
+const RegisterToVote = () => {
+  return (
+    <div className="register-to-vote-form">
+      <iframe className="register-to-vote-i-frame"
+        src="https://register.vote.org/?partner=111111&campaign=free-tools" 
```

Comment:
I just did a little test, and if you set `src="https://register.vote.org/es?campaign=free-tools&partner=111111"` (es in front of the ?), then it displays in Spanish! 

If the user picks Spanish as their language for emails (or they select the button in the header), then the state of this iframe src attribute changes to include `es?` ? 

Just a little worm for your brain!  😊